src/test/java/org/opentripplanner/apis/gtfs/mapping/LocalDateRangeMapperTest.java

```diff
@@ -0,0 +1,42 @@
+package org.opentripplanner.apis.gtfs.mapping;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opentripplanner.apis.gtfs.generated.GraphQLTypes;
+import org.opentripplanner.apis.gtfs.support.time.LocalDateRangeUtil;
+
+class LocalDateRangeMapperTest {
+
+  private static final LocalDate DATE = LocalDate.parse("2024-05-27");
+
+  public static List<GraphQLTypes.GraphQLLocalDateRangeInput> noFilterCases() {
```

Comment:
```suggestion
 private static List<GraphQLTypes.GraphQLLocalDateRangeInput> noFilterCases() {
```

Reply:
Why is this marked as resolved?

Reply:
To be honest, I don't remember since it was a while ago. I thought that the methods for the `@MethodSource` must be at least package-private (but cannot be private) but I was wrong. I made it private in 80fedfa9275e046de3583bdc52077ccefb0cc963

Reply:
If it need to be package-private that is fine, but if a method is `public` it means it is used elsewhere - which again indicate that basic test principles are not followed. This is not the only one ...

Reply:
I think this happened when I made Intellij generate the method stub for the cases. That's my bad. I will see if I can reconfigure Intellij to not make the method not public.